packages/cms-cli/commands/hubdb.js

```diff
@@ -71,6 +77,43 @@ function configureHubDbCreateCommand(program) {
   addConfigOptions(program);
 }
 
+function configureHubDbImportCommand(program) {
+  program
+    .version(version)
+    .description('Import HubDB table rows')
+    .arguments('<tableId> <src>')
+    .action(async (tableId, src, command = {}) => {
+      setLogLevel(command);
+      logDebugInfo(command);
+      const { config: configPath } = command;
+      loadConfig(configPath);
+      checkAndWarnGitInclusion();
+
+      if (!(validateConfig() && (await validatePortal(command)))) {
+        process.exit(1);
+      }
+      const portalId = getPortalId(command);
+
+      try {
+        const table = await importHubDbTableRows(
+          portalId,
+          tableId,
+          path.resolve(getCwd(), src)
+        );
+        logger.log(
+          `The table ${table.tableId} was updated in ${portalId} with ${table.rowCount} rows`
+        );
+      } catch (e) {
+        logger.error(`Updating the table at "${src}" failed`);
+        logger.error(e.message);
```

Comment:
We have handlers to log `Error` instances.
Example here https://github.com/HubSpot/hubspot-cms-tools/blob/d98f0449e0add7dc37205d1c8b614f6cb66256d1/packages/cms-cli/commands/lint.js#L11